test/jdk/java/lang/reflect/Proxy/DefaultMethods.java

```diff
@@ -322,33 +322,31 @@ public void testIllegalArgument(Class<?>[] interfaces, Class<?> defc, String nam
         }
     }
 
-    @DataProvider(name = "arguments")
-    private Object[][] arguments() {
+    @DataProvider(name = "illegalArguments")
+    private Object[][] illegalArguments() {
         return new Object[][] {
-                new Object[] {},
-                new Object[] { 100, "foo", 100 },
-                new Object[] { "foo", 100},
+            new Object[] {},
```

Comment:
I'm okay with using 4-space indentation here.  Would you mind changing other data providers to use the same format consistently?

Reply:
Ok.

Reply:
I added a commit for indentation and new test case in test. I can't directly push this to your repository:

> ERROR: Permission to mlchung/jdk.git denied to plevart

But you can merge this PR into your branch and the result will be the same: Your PR against jdk/master will be updated with merged changes from my branch.